sbin/build.sh

```diff
@@ -565,6 +565,19 @@ removingUnnecessaryFiles() {
     rm -rf "${dirToRemove}"/demo || true
   fi
 
+  # Generate java.se adopt bundle
+  moduleList="java.se"
```

Comment:
It will be interesting to see what else we need here.  I assume you'll do some experiments?  Trying a desktop app that connects to things is always good.  Also JavaFX

Reply:
yeah trial and error is needed here

Reply:
You can do `—list-modules` on the JRE AFAIK which should you give what is needed. Then add jdk.unsupported.desktop on top for OpenJFX.